internal/move2kubeapi/handlers/inputs.go

```diff
@@ -53,6 +53,15 @@ func HandleCreateProjectInput(w http.ResponseWriter, r *http.Request, isCommon b
 	}
 	r.Body = http.MaxBytesReader(w, r.Body, common.Config.MaxUploadSize)
 	if err := r.ParseMultipartForm(common.Config.MaxUploadSize); err != nil {
+		if _, ok := err.(*http.MaxBytesError); ok {
+			logrus.Errorf("request body exceeded max upload size of '%d' bytes. Error: %q", common.Config.MaxUploadSize, err)
```

Comment:
Would it be good to mention which config to change?

Reply:
done